File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/database/statistikk/DeleteSykefraværsstatistikkFunction.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.database.statistikk;

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene.ÅrstallOgKvartal;

@FunctionalInterface
public interface DeleteSykefraværsstatistikkFunction {
  int apply(ÅrstallOgKvartal årstallOgKvartal);
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/database/statistikk/DeleteSykefraværsstatistikkFunction.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.database.statistikk

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller.ÅrstallOgKvartal

fun interface DeleteSykefraværsstatistikkFunction {
    fun apply(årstallOgKvartal: ÅrstallOgKvartal): Int
}
